FluidNC/data/maslow.yaml

```diff
@@ -1,10 +1,20 @@
 board: Maslow
 name: Maslow S3 Board
 
-Maslow_frame_width: 3000.0
-Maslow_frame_height: 2000.0
-Maslow_vertical: 1
+Maslow_vertical: false
+Maslow_calibration_offset: 1000
 
+Maslow_blX: 0.0
```

Comment:
Do these persist through power cycling now?

Reply:
Yep

Reply:
I just tried and they don't for me 😛